_data/events_gdcr2019/London_Codurance.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "GDCR19 in London (@Codurance)",
+  "url": "https://www.meetup.com/london-software-craftsmanship/events/265499324/",
```

Comment:
The link doesn't work :cry:

Reply:
Please, check it here: https://deploy-preview-557--coderetreat-org.netlify.com/events/

Reply:
Oh, I have the entry in preview mode. I was waiting for this to be merged before making it public. It should be visible now